fix(consensus): fix the problem when setting the genesis.json file consensus_params.block.max_gas=-1 (#1740)
evmos/evmos: CHANGELOG.md

- (ante) [#1693](https://github.com/evmos/evmos/pull/1693) Prevent panic on int64 conversion in EVM fees antehandler.
- (evm) [#1693](https://github.com/evmos/evmos/pull/1703) Prevent panic on uint64 conversion in EVM keeper `ApplyMessageWithConfig` function.
- (deps) [#1718](https://github.com/evmos/evmos/pull/1718) Update rosetta types import.

> - (consensus) [#1740](https://github.com/evmos/evmos/pull/1740) Fix the problem when setting the genesis.json file consensus_params.block.max_gas=-1.

Reviewer: ```suggestion
- (consensus) [#1740](https://github.com/evmos/evmos/pull/1740) Enable setting block gas limit to max by specifying it as -1 in the genesis file.
```
